### ISSUE TITLE
Binary Cross Entropy loss for multi-label classification

### ISSUE THREAD

**Issue by acadTags** [ThilinaRajapakse/simpletransformers#101]
Great thanks for sharing this easy-to-use implementation. May I ask one question here?

It seems that the multilabel loss is the softmax cross-entropy.
https://github.com/ThilinaRajapakse/simpletransformers/blob/04886b56c2b9ead0569a2f8c5338631233ce22a6/simpletransformers/classification/transformer_models/bert_model.py#L68

Can we use the binary cross-entropy loss with sigmoid activation? Like this one
[```torch.nn.functional.binary_cross_entropy_with_logits```](https://pytorch.org/docs/1.0.0/nn.html?highlight=binary_cross_entropy_with_logits#torch.nn.functional.binary_cross_entropy_with_logits
)

Best wishes,
acadtags

**Comment by ThilinaRajapakse** [comment]
Multilabel classification uses different classes from other classification tasks. You can find them here.

https://github.com/ThilinaRajapakse/simpletransformers/blob/04886b56c2b9ead0569a2f8c5338631233ce22a6/simpletransformers/custom_models/models.py#L60

Multilabel does use BCE rather than CrossEntropy.

**Comment by acadTags** [comment]
> Multilabel classification uses different classes from other classification tasks. You can find them here.
> 
> https://github.com/ThilinaRajapakse/simpletransformers/blob/04886b56c2b9ead0569a2f8c5338631233ce22a6/simpletransformers/custom_models/models.py#L60
> 
> Multilabel does use BCE rather than CrossEntropy.

Now I got it, thanks!
